### ISSUE TITLE
Error on Score All using image classification

### ISSUE THREAD

**Issue by braymp** [CellProfiler/CellProfiler-Analyst#117]
Using the attached properties file and training set on e2c5064d104e7d0f5f4199e360a085d794ddb694,  training the classifier with the defaults (random forest, 5 features), and then Score All produces the following error:
```
￼ File "C:\Trunk\pull_requests\CellProfiler-Analyst\CellProfiler-Analyst.py", line 531, in <module>
  app.MainLoop()
File "C:\Python27\Lib\site-packages\wx-3.0-msw\wx\_core.py", line 8657, in MainLoop
  wx.PyApp.MainLoop(self)
File "C:\Python27\Lib\site-packages\wx-3.0-msw\wx\_core.py", line 7952, in MainLoop
  return _core_.PyApp_MainLoop(*args, **kwargs)
￼ File "C:\Trunk\pull_requests\CellProfiler-Analyst\cpa\classifier.py", line 1547, in ScoreAll
  self.keysAndCounts = self.algorithm.PerImageCounts(number_of_classes, filter, update)
￼ File "C:\Trunk\pull_requests\CellProfiler-Analyst\cpa\generalclassifier.py", line 109, in PerImageCounts
  return multiclasssql.PerImageCounts(self, number_of_classes, filter_name, cb)
￼ File "C:\Trunk\pull_requests\CellProfiler-Analyst\cpa\multiclasssql.py", line 267, in PerImageCounts
  counts = do_by_steps(p.object_table, filter_name, p.area_scoring_column)
￼ File "C:\Trunk\pull_requests\CellProfiler-Analyst\cpa\multiclasssql.py", line 252, in do_by_steps
  predicted_classes = classifier.Predict(cell_data)
￼ File "C:\Trunk\pull_requests\CellProfiler-Analyst\cpa\generalclassifier.py", line 113, in Predict
  predictions = self.classifier.predict(test_values)
File "C:\Python27\Lib\site-packages\sklearn\ensemble\forest.py", line 498, in predict
  proba = self.predict_proba(X)
File "C:\Python27\Lib\site-packages\sklearn\ensemble\forest.py", line 537, in predict_proba
  X = self._validate_X_predict(X)
File "C:\Python27\Lib\site-packages\sklearn\ensemble\forest.py", line 319, in _validate_X_predict
  return self.estimators_[0]._validate_X_predict(X, check_input=True)
File "C:\Python27\Lib\site-packages\sklearn\tree\tree.py", line 365, in _validate_X_predict
  X = check_array(X, dtype=DTYPE, accept_sparse="csr")
File "C:\Python27\Lib\site-packages\sklearn\utils\validation.py", line 373, in check_array
  array = np.array(array, dtype=dtype, order=order, copy=copy)

ValueError: could not convert string to float: i
```
[saturated_training_set.txt](https://github.com/CellProfiler/CellProfiler-Analyst/files/92488/saturated_training_set.txt)
[2014_09_09_GastricCancer_AdamBass_qc_2015_09_29_try2.properties.txt](https://github.com/CellProfiler/CellProfiler-Analyst/files/92489/2014_09_09_GastricCancer_AdamBass_qc_2015_09_29_try2.properties.txt)

**Comment by jhung0** [comment]
should be fixed

**Comment by braymp** [comment]
Nope, same problem appears with 144cd2b609849c3d5514090112d3463dbdcfa0fb

**Comment by jhung0** [comment]
Ok hopefully it's fixed 361b7fb80a54f63d286cb60da67418025fa52891

**Comment by braymp** [comment]
Yep, looks like that did it.
